lib/Serialization/Serialization.cpp

```diff
@@ -1032,10 +1032,12 @@ void Serializer::writeInputBlock(const SerializationOptions &options) {
   ImportSet spiImportSet =
       getImportsAsSet(M, ModuleDecl::ImportFilterKind::SPIAccessControl);
 
+  // MARK: need to fix the types of allImports and bridgingHeaderImport
```

Comment:
Not sure what this comment is trying to indicate 😅? Have you fixed this/is this a TODO?

Reply:
I took out the comment--it was left over from a thought I was processing.